### ISSUE TITLE
ipad status bar mod little11 issues facebook comment pages

### ISSUE THREAD

**Issue by roipurwanto** [ryannair05/Little11#43]
little11 with ipad status bar mod conflict with scrolling facebook comment page.
status bar going blinked and stuck facebook comment page when scrolling the page.

**Comment by dinouse** [comment]
Im using ipadstatusbar tweaks + little11 standard bar. it work like charm!

**Comment by nguyenthienhoan** [comment]
> little11 với xung đột mod thanh trạng thái ipad với cuộn trang bình luận facebook.
> thanh trạng thái sẽ nhấp nháy và bị kẹt trang bình luận facebook khi cuộn trang.

Exactly, use iPad or iP X Startusbar animation with Screen Corner so facebook will be crash if you doesnt use bottom bar. But MTAC can be do that, i will contact him soon. 

> Tôi đang sử dụng tinh chỉnh ipadstatusbar + thanh chuẩn11. Nó làm việc như quyến rũ!

Your ipadstatusbar just enable iPad statusbar and doesnt work with Screen Corner. So that working very good.

**Comment by dinouse** [comment]
this is solved with latest version
